internal/broker/amqp/consumer.go

```diff
@@ -104,13 +104,15 @@ func (c *consumer) Start(logger *log.Logger, handlers map[string]func([]byte) er
 				"status":       "failed",
 				"responseTime": duration,
 				"error":        fmt.Sprintf("%+v", err),
-			}).Errorf("[consumer] [FAILED] Failed to handle message: nacking and requeing: %v", err)
+			}).Errorf("[consumer] [FAILED] Failed to handle message: trigger error handling and acking: %v", err)
 			// TODO: remove comments to allow for redelivery. This will put events
 			// into the unacknowledged state
 
 			// err := msg.Nack(false, true) if err != nil {
 			//  logger.WithFields("error", fmt.Sprintf("%+v", err)).Errorf("Failed to nack message: %v", err)
 			// }
+			errorHandler(msg.Type, msg.Body, err)
+			msg.Ack(false)
```

Comment:
Actually I think we should Nack with requeue=false instead to keep AMQP semantics intact. It is the same outcome but lets us track these failure handlings in existing metrics

Reply:
Sounds good. Wasn't totally sure of the outcome of that nack, but that makes sense 👍